### PR TITLE
[nginx] replication headers and configurations

### LINKED ISSUE #1: [nginx] configuration
Set correct headers for openwisp configurations by replicating the configurations of the ansible-openwisp2.

Ensure following are enabled by default but can be turned off if needed:
- Gzip
- HTTP/2
- IPv6

Blocks: #7 , #2

### PR DESCRIPTION
**Pull request by atb00ker**
- Removed openwisp-orchestration
- Replicating the ansible-openwisp nginx settings
without "sdpy", because sdpy has been superseded
by http/2 and nginx doesn't provide it by default.

Closes #1

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -34,10 +34,12 @@ TOPOLOGY_APP_SERVICE=topology
 NGINX_HTTP2=http2
 NGINX_CLIENT_BODY_SIZE=5M
 NGINX_IP6=True
-NGINX_GZIP=off
+NGINX_ADMIN_ALLOW_NETWORK=all
+NGINX_SSL_CONFIG=gzip on;gzip_comp_level 6;gzip_proxied any;gzip_min_length 1000;gzip_types text/plain image/svg+xml application/json application/javascript text/xml text/css application/xml application/x-font-ttf font/opentype;
```
Comment (nemesisdesign) [inline .env]: how does this work?
is it going to be expanded? Or it doesn't need to be on different new lines?
Wouldn't something like this be hard to manage?

Comment (nemesisdesign) [reply .env]: Maybe you can create an issue to improve this later as well

**Comment by nemesisdesign** [comment]
PS: PR is flagged as draft

File: .env (hunk 1 above)
Comment (atb00ker) [reply .env]: It doesn't need to be expanded. I can move them to seperate variables if needed.
Please note that even right now, in the ansible-openwisp2 I see the implementation of gzip is in the same line.
([screenshot here](https://user-images.githubusercontent.com/24985760/58369162-411cf100-7f14-11e9-9d65-6f2b59a1091c.png).)

**Comment by atb00ker** [comment]
> PS: PR is flagged as draft

Will change that soon and ping back. :-)

File: build/openwisp_nginx/global.conf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Global settings across valid across all
```
Comment (2stacks) [inline build/openwisp_nginx/global.conf]: Fix wording.

File: build/openwisp_nginx/global.conf [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+# configration files
+ 
+server_names_hash_bucket_size $NGINX_SERVER_NAME_HASH_BUCKET;
```
Comment (2stacks) [inline build/openwisp_nginx/global.conf]: At first read this gives me the impression that this file could contain NGINX global configs.  However, everything in conf.d get's nested at the top of the `http {}` context.  Not sure it matters but the name global may be confusing.

File: build/openwisp_nginx/global.conf (hunk 2 above)
Comment (atb00ker) [reply build/openwisp_nginx/global.conf]: Changed to `# Custom http block configurations`

File: build/openwisp_nginx/global.conf (hunk 3 above)
Comment (atb00ker) [reply build/openwisp_nginx/global.conf]: I agree. Changed file references to `http.conf`. :-)

File: .env (hunk 1 above)
Comment (nemesisdesign) [reply .env]: the resulting file is not very relevant, what is relevant is the format we use to keep the variables, we need something a bit more flexible, having every value on one var would be painful.

I suggest creating an issue for this specific thing, we can investigate later how to improve it

File: README.md [hunk 4]
```diff
@@ -102,4 +102,4 @@ Note:
    - Default username & password are `admin`.
    - Default domains are: dashboard.openwisp.org, controller.openwisp.org, radius.openwisp.org and topology.openwisp.org.
    - You may want to add the domains in your hosts file, command: `echo "127.0.0.1 dashboard.openwisp.org controller.openwisp.org radius.openwisp.org topology.openwisp.org" >> /etc/hosts/`
-   - You may want to change `createProdCerts` to `createDevCerts` in the `build/openwisp_orchestration/init_command.sh` file for development to get self-signed certificates.
+   - You may want to change `create_prod_certs` to `create_dev_certs` in the `build/openwisp_orchestration/init_command.sh` file for development to get self-signed certificates.
```
[the remark this answers is not in the document]
Comment (atb00ker) [reply README.md]: Yes. Hence, this line has been deleted. This comment seems on a previous commit. If you refresh the page, this has been removed already.

**Pull request merged**